x-pack/plugins/spaces/server/lib/utils/space_solution_disabled_features.test.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import type { KibanaFeature } from '@kbn/features-plugin/server';
+
+import { withSpaceSolutionDisabledFeatures } from './space_solution_disabled_features';
+
+const features = [
```

Comment:
Can we add features in other categories, to verify that we don't accidently remove them?

Reply:
Good idea, addressed in https://github.com/elastic/kibana/pull/191927/commits/ea20d7104dd11c056947123cb343546fdf5e95fe